packages/generator/src/edmx-parser/common/edmx-types.ts

```diff
@@ -30,7 +30,7 @@ export interface EdmxDocumented {
 
 export interface EdmxParameter extends EdmxDocumented, EdmxNamed {
   Type: string;
-  Nullable?: string;
+  Nullable: string;
```

Comment:
Is this optional?

Reply:
In the edmx parser I always set the value to the default `true`